src/content/guides/production.md

```diff
@@ -236,9 +237,33 @@ __src/index.js__
   document.body.appendChild(component());
 ```
 
+## Module Concatenation
+
+By default, webpack wraps each module into a function. This helps helps to isolate them, but brings additional overhead. You can try reducing this overhead by enabling [`ModuleConcatenationPlugin`](/plugins/module-concatenation-plugin) which, where possible, merges multiple modules together in a single module wrapper:
+
+__webpack.prod.js__
+
+```diff
++ const webpack = require('webpack');
+  const merge = require('webpack-merge');
+  const common = require('./webpack.common.js');
+
+  module.exports = merge(common, {
+    plugins: [
++     new webpack.optimize.ModuleConcatenationPlugin()
+    ]
+  })
+```
```

Comment:
This example should be a true diff by including the full content of the previous __webpack.prod.js__ example, e.g.

``` diff
  const webpack = require('webpack');
  const merge = require('webpack-merge');
  const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
  const common = require('./webpack.common.js');

  module.exports = merge(common, {
    devtool: 'cheap-module-source-map',
    plugins: [
      new UglifyJSPlugin({
        sourceMap: true
      }),
      new webpack.DefinePlugin({
        'process.env': {
          'NODE_ENV': JSON.stringify('production')
        }
-     })
+     }),
+     new webpack.optimize.ModuleConcatenationPlugin()
    ]
  })
```

Reply:
Done!

Instead of doing

```diff
-     })
+     }),
```

I added a comma in the `DefinePlugin` section.